parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -87,6 +87,30 @@ impl Command {
 			fun: Fungible(buy_execution_fee_amount),
 		};
 
+		let create_instructions = |origin_location: Junction| -> Vec<Instruction<()>> {
+			vec![
+				UniversalOrigin(GlobalConsensus(network)),
+				DescendOrigin(X1(origin_location)),
+				WithdrawAsset(buy_execution_fee.clone().into()),
+				BuyExecution { fees: buy_execution_fee.clone(), weight_limit: Unlimited },
+				SetAppendix(
+					vec![
+						RefundSurplus,
+						DepositAsset {
+							assets: buy_execution_fee.into(),
```

Comment:
Just reminder there is one fix here in https://github.com/Snowfork/snowbridge/pull/952

Reply:
@yrong cool thanks for the reminder. Can I merge this and then you update https://github.com/Snowfork/snowbridge/pull/952?